tasks/cea-install.js

```diff
@@ -58,15 +58,9 @@ const checkTargetDir = async () => {
 
 // Create new package.json
 const npmInit = async () => {
-  const appPkg = {
-    name: TARGET_DIR,
-    version: '0.1.0',
-    private: true
-  };
+  let appPkg = templatePkg;
 
-  appPkg.scripts = templatePkg.scripts;
-  appPkg.dependencies = templatePkg.dependencies;
```

Comment:
I don't think this is a good idea. Because now the bin, author, homepage, are included from the template

Reply:
Take a look at the contents of this file and the changes I made there, none of those fields exist anymore in _template/package.json_ 👀 
https://github.com/ProjectEvergreen/create-evergreen-app/blob/bug/issue-82-npm-init-order/template/package.json

Reply:
I thought this was one of the values of splitting this all up, right?  Two package.json's, one for the actual CEA package, and one for the CEA output.